### PR TITLE
Add Usage Example to Keras Nadam Optimizer

### PR DESCRIPTION
**Pull request by jedlimlx**

### CODE REVIEW

File: tensorflow/python/keras/optimizer_v2/nadam.py [hunk 1]
```diff
@@ -59,6 +59,17 @@ class Nadam(optimizer_v2.OptimizerV2):
 
   References
     See [Dozat, T., 2015](http://cs229.stanford.edu/proj2015/054_report.pdf).
+    
+  Usage Example:
```
Comment (boronhub) [inline tensorflow/python/keras/optimizer_v2/nadam.py]: backtick ` ``` `  notation isn't needed.

File: tensorflow/python/keras/optimizer_v2/nadam.py [hunk 2]
```diff
@@ -59,6 +59,17 @@ class Nadam(optimizer_v2.OptimizerV2):
...
+    ```python
```
Comment (boronhub) [inline tensorflow/python/keras/optimizer_v2/nadam.py]: Please remove `import tensorflow as tf`, as that import is predefined.

**Review by boronhub — CHANGES_REQUESTED** [review]

**Review by generationzcode — CHANGES_REQUESTED** [review]
Hey, nice example, can you just add some code that will give an output then define what output it will give so that it runs as a doctest?🖖

**Comment by generationXcode** [comment]
Hi thats nice, but the output doesn't show anything about the optimizer, can you try and give an output on something related to the optimizer?

**Comment by generationXcode** [comment]
awesome!

**Review by generationzcode — DISMISSED** [review]

**Comment by jedlimlx** [comment]
@boronhub @generationXcode Can someone take a look at the pull request? I made some changes as build on Linux CPU had failed.

**Comment by gbaned** [comment]
@jedlimlx Can you please resolve conflicts? Thanks!

File: tensorflow/python/keras/optimizer_v2/nadam.py [hunk 3]
```diff
@@ -59,6 +59,18 @@ class Nadam(optimizer_v2.OptimizerV2):
 
   References
     See [Dozat, T., 2015](http://cs229.stanford.edu/proj2015/054_report.pdf).
+    
+  Usage Example:
```
Comment (tanzhenyu) [inline tensorflow/python/keras/optimizer_v2/nadam.py:63]: I would really like to see examples like this:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/optimizer_v2/adam.py#L73-L79

**Comment by gbaned** [comment]
@jedlimlx Can you please check @tanzhenyu's comments and resolve conflicts?. Thanks!

**Review by rthadur — APPROVED** [review]

**Review by rthadur — APPROVED** [review]

File: tensorflow/python/keras/optimizer_v2/nadam.py [hunk 4]
```diff
@@ -51,6 +50,14 @@ class Nadam(optimizer_v2.OptimizerV2):
       `"clipnorm"` (float) clips gradients by norm; `"clipvalue"` (float) clips
       gradients by value.
 
+  Usage Example:
+    >>> opt = tf.keras.optimizers.Nadam(learning_rate=0.2)
+    >>> var1 = tf.Variable(10.0)
+    >>> loss = lambda: (var1 ** 2) / 2.0
+    >>> step_count = opt.minimize(loss, [var1]).numpy()
+    >>> var1.numpy()
+    9.8
```
Comment (rthadur) [inline tensorflow/python/keras/optimizer_v2/nadam.py:59]: Getting this error internally , @jedlimlx can you please check 

```
tensorflow.python.keras.optimizer_v2.nadam.Nadam
Failed example:
    var1.numpy()
Expected:
    9.8
Got:
    9.78871
```

**Review by rthadur — CHANGES_REQUESTED** [review]

**Review by tanzhenyu — APPROVED** [review]

**Pull request merged**